### ISSUE TITLE
An invalid AWS configuration provided in the `awsConfig` field should not cause the Foundry VTT process to crash on startup.

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/foundryvtt#9322]
https://discord.com/channels/170995199584108546/1065764070680186930/1101513138408538152

Stacktrace:
```
/home/foundry/foundry-testing-install/resources/app/node_modules/@aws-sdk/credential-provider-node/dist-cjs/defaultProvider.js:13
    throw new property_provider_1.CredentialsProviderError("Could not load credentials from any providers", false);
          ^

CredentialsProviderError: Could not load credentials from any providers
    at /home/foundry/foundry-testing-install/resources/app/node_modules/@aws-sdk/credential-provider-node/dist-cjs/defaultProvider.js:13:11
    at /home/foundry/foundry-testing-install/resources/app/node_modules/@aws-sdk/property-provider/dist-cjs/chain.js:11:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async coalesceProvider (/home/foundry/foundry-testing-install/resources/app/node_modules/@aws-sdk/property-provider/dist-cjs/memoize.js:14:24)
    at async Object.credentialDefaultProvider (/home/foundry/foundry-testing-install/resources/app/node_modules/@aws-sdk/property-provider/dist-cjs/memoize.js:33:24) {
  tryNextLink: false
}
```

Logged Options:
```json
{
  "updateChannel": "prototype",
  "upnp": false,
  "upnpLeaseDuration": null,
  "awsConfig": "******/aws.json",
  "passwordSalt": null,
  "sslCert": null,
  "sslKey": null,
  "world": null,
  "serviceConfig": null,
  "adminPassword": "••••••••••••••••",
  "compressSocket": true,
  "cssTheme": "foundry",
  "hotReload": false
}
```

`aws.json`:
```json
{
  "accessKeyId": "DO00M9E387GW7UB72ZCT",
  "secretAccessKey": "<REDACTED>",
  "region": "eu-west-1",
  "endpoint": "fra1.digitaloceanspaces.com"
}
```

**Comment by Fyorl** [comment]
Not a bug, the format of the aws config file has changed in SDK 3, as per #8898

**Comment by aaclayton** [comment]
Tentatively re-opening because I think this error should be caught and surfaced to the client rather than something that causes the Foundry process to crash.

**Comment by alancnet** [comment]
The documentation at https://foundryvtt.com/article/aws-s3/ does not reflect the new JSON structure.

**Comment by Doresain** [comment]
+1 re: the out of date documentation.

It also seems like this crash was not solved - installing 11.302 caused the Foundry process to crash on startup until I updated to the newer json structure.

**Comment by kakaroto** [comment]
Updated to 11.304 and it doesn't open locally anymore. Seems to crash immediately after it starts.
![image](https://github.com/foundryvtt/foundryvtt/assets/27990/72b6be26-c1e8-451c-a541-d9253e7ba1e8)

Looks like a regression of this issue
